sensors/ev3_uart_sensor_defs.c

```diff
@@ -35,7 +35,7 @@ const struct ev3_uart_sensor_info ev3_uart_sensor_defs[] = {
 		.mode_info	= {
 			[0] = {
 				/**
-				 * @description: Reflected light<br />Red LED on
+				 * @description: Reflected light<br />LED color: red
```

Comment:
Can we make the LED color descriptions consistent?

I think it is better with a colon (`:`) like you have used here, but it does not match the other descriptions. So either add a colon to the rest of leave this as it was.